src/LibChorusTests/VcsDrivers/Mercurial/RepositoryTests.cs

```diff
@@ -140,18 +140,22 @@ public void RepositoryRecoversFromIncompleteMerge()
 		}
 
 		[Test]
-		public void CloneToUsbWithoutUpdate_CreatesExtensionSectionInHgrc()
+		public void CloneToUsbWithoutUpdateFollowedByIdentifierDoesNotAffectHgrc()
 		{
 			using(var repo = new RepositorySetup("source"))
 			using(var f = new TemporaryFolder("clonetest"))
 			{
 				// The MakeCloneFromLocalToLocal with false on alsoDoCheckout is the core of the usb clone operation.
-				// We need to make sure that the extensions are used in the clone
-				HgHighLevel.MakeCloneFromLocalToLocal(repo.ProjectFolder.Path, f.Path, false, new NullProgress());
+				// We need to make sure that this clone is bare of extensions, and remains so after the identifier is checked.
```

Comment:
This sounds as if you will force the clone not to have any extensions at all, rather than just being careful not to change anything, which sounds more reasonable for an identity check. Is that really what you mean?